app/views/alchemy/pages/show.rb

```diff
@@ -131,6 +132,22 @@ def google_analytics_code
         ENV.fetch('GOOGLE_ANALYTICS_CODE', 'UA-123456-1')
       end
 
+      def as_json
+        {
+          url: page_object.url,
+          credit_image: page_object.credit_image,
+          description: page_object.description,
+          full_image: page_object&.media&.first&.dig(:image, :full, :url),
+          card_image: page_object&.media&.first&.dig(:image, :thumbnail, :url),
```

Comment:
To what does "card" refer here? To a Portal promo card? If so, rename these keys to be descriptive outside of that use case.

Reply:
It's the foyer cards too, currently editors (re-)upload a cropped version for each. There was talk of in future automatically generating them from an exhibition's content. In that case they could load these in via AJAX perhaps and use this code here also.

Reply:
It's the foyer cards too, currently editors (re-)upload a cropped version for each. There was talk of in future automatically generating them from an exhibition's content. In that case they could load these in via AJAX perhaps and use this code here also.